www/src/config.ts

```diff
@@ -139,6 +169,12 @@ export const SIDEBAR_HEADER_MAP: Record<
   Exclude<KnownLanguageCode, "en">,
   Record<OuterHeaders, string>
 > = {
+  // Translate the sidebar's "outer headers" here
+  ar: {
+    "Create T3 App": "إنشاء تطبيق T3.",
+    Usage: "كيفية الإستخدام؟",
+    Deployment: "نَشر مَوقعك",
+  },
   // Translate the sidebar's "outer headers" here
```

Comment:
can you move this comment to the top of the object and delete the duplicate line?